src/test/crimson/cbt/radosbench_4K_read.yaml

```diff
@@ -7,14 +7,14 @@ tasks:
 - cbt:
     benchmarks:
       radosbench:
-        concurrent_ops: 16
-        concurrent_procs: 2
+        concurrent_ops: 64
```

Comment:
@xxhdx1985126 why does more concurrent ops help to increase the reliability of the perf test?

@rzarzynski what do you think?

Reply:
I just wanted to make sure that enough pressure is created, otherwise the perf result may not represent the throughput the osds can provide

Reply:
@xxhdx1985126 

i thought it's was to make the test result more statistical significant by prolonging the test time.

but it seems that you want to put more load on OSD to demonstrate the throughput that crimson is able to offer. this is nice to have when comparing crimson osd with classic osd, so we can understanding the advantage of crimson osd in this perspective. but why do we need to do so when checking if a change that could hurt performance?

Reply:
I think if we don't push crimson to the limit, some performance degradation may not be easily shown. Say the crimson can offer iops of 50000, if only push 40000 ops per second, the performance degradation may not be observed through the result unless it's more than 25%. Am I right?